examples/plot_spm_auditory.py

```diff
@@ -1,37 +1,116 @@
 """
```

Comment:
Should I put back all the scripts in the examples directory, removing the subdirectories I created? I did this as an attempt to sort the examples in a meaningful order on the documentation page, but this may be a bad idea when you integrate nistats to nilearn. Just tell me.  
PS: sorry for the images, a mismanipulation from my part.